gst/nnstreamer/tensor_query/tensor_query_common.h

```diff
@@ -18,42 +18,72 @@
 #include <gst/gst.h>
```

Comment:
TODO : glib, gst, and tensor_common.h
remove these and use std only (e.g. tensor_typedef.h is ok)

Reply:
Thanks. Removed!